dashboard/app/models/pd/teacher_application.rb

```diff
@@ -96,6 +96,11 @@ def application_json
     application
   end
 
+  # Convenience method to set value(s) on the application JSON
+  def update(update_hash)
```

Comment:
I just realized I shouldn't call this update, because it will override ActiveRecord's [update](http://apidock.com/rails/ActiveRecord/Persistence/update). I'll rename to update_application_hash